### ISSUE TITLE
[Web Component]: Alert Box - The component should have a Boolean prop like "assertive" to modify the live region

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#23793]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

The `div[aria-live="polite"]` container should have a prop like "assertive" that if set to True, would change the aria-live value to "assertive". This causes the screen reader behavior to interrupt any text being spoken, instead of waiting for the current buffer to be emptied before speaking the live region. There are times this is a necessary practice, such as a form submission error.

NOTE: This feature can be added in the near future if there's a need to ship the component, because it has some nuance to how we handle it.

There are two key pieces of information developers and teams using the component need to be aware of:

* This "assertive" prop should also add a `role="alert"` to the containing div.
* These alert messages that interrupt the screen reader's natural flow will have to be added dynamically. Live regions do not reliably read information on first page load, only after something has changed.
  * Can we explore something like dynamic imports or a `setTimeout()` to load a container, then load the alert box 250ms later or some short period that will cause screen readers to announce it?
* `role="alert"` does NOT work across the board when the component is on the page on first paint. See https://codepen.io/tpierce_402/pen/GRorBrL

## Acceptance Criteria

- [ ] Storybook is updated with a new story for an alert box that can be set to assertive
- [ ] Assertive alert boxes announce to screen readers properly
- [ ] Zero axe-core violations detected

**Comment by caw310** [comment]
Closing as this is old and the VADS has been updated. If there is still an issue file a new bug.
